Sources/VLCMovieViewController.m

```diff
@@ -933,7 +935,7 @@ - (void)deviceMotionHasAttitudeWithDeviceMotion:(VLCDeviceMotion *)deviceMotion
 }
 #pragma mark - controls
 
-- (IBAction)closePlayback:(id)sender
+- (void)closePlayback:(id)sender
```

Comment:
Why is this changed? I can't find the call to `closePlayback`, this is probably fine, just double-checking.

Reply:
it is still added in code but it is not anymore connected through the xib